packages/cache-url/package.json

```diff
@@ -42,6 +42,8 @@
   },
   "devDependencies": {
     "@ampproject/rollup-plugin-closure-compiler": "0.8.2",
+    "@babel/core": "^7.1.2",
+    "babel-plugin-filter-imports": "^2.0.3",
```

Comment:
This project appears to rely on specific versions of dependencies, can we pin here instead of using hat versions?